docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -0,0 +1,211 @@
+
+General Syntax
+
+* Conditions can use *AND*, *OR*, *NOT* and *()*
+
+* Conditions can use *==*, *&lt;*, *&gt;*, *&gt;=*, *&lt;=* to compare
+
+* Number values can be referred to by Hex (0xabc123) or Decimal (10000000000)
+
+* String values can only be compared via *==*
+
+* Includes basic math operators: *+*, *-*, ***, */*, *^*
+
+=== Transaction Conditions
+
+IMPORTANT: If a transaction condition is specified, then a transaction MUST meet this condition in order to trigger a notification.
+
+NOTE: To accomodate comparisons for checksum and non-checksum addresses, expressions are case-insensitive.
+
+Transaction Conditions can refer to the following properties
+
+* *to* is the _to_ address for the transaction
+
+* *from* is the _from_ address for the transaction
+
+* *gasPrice* is the price of gas sent in the transaction
+
+* *gasUsed* is the amount of gas used in the transaction
+
+* *value* is the value sent in the transaction
+
+* *nonce* is the nonce for the specific transaction
+
+* *status* is a derived value and can be compared with *"success"* or *"failed"*
+
+==== Example Conditions
+
+Transactions that are reverted
+
+[source,jsx]
+----
+status == "failed"
+----
+
+Transactions excluding those from 0xd5180d374b6d1961ba24d0a4dbf26d696fda4cad
+
+[source,jsx]
+----
+from != "0xd5180d374b6d1961ba24d0a4dbf26d696fda4cad"
+----
+
+Transactions that have BOTH a gasPrice higher than 50 gwei AND a gasUsed higher than 20000
+
+[source,jsx]
+----
+gasPrice > 50000000000 and gasUsed > 20000
+----
+
+=== Event and Function Conditions
+
+Event and Function conditions further narrow the set of transactions that trigger notification.  These can refer to arguments in the signature either by name (if argument is named) or by index (e.g., $0, $1...).  The variables that are available to you are indicated in the user interface as you specify these functions.
+
+==== Example Conditions
+
+Transactions that emit a Transfer(...) event with a value between 1 and 100 ETH (in hex)
+
+[source,jsx]
+----
+// Event Signature: Transfer(address to, address from, uint256 value)
+value > 0xde0b6b3a7640000 and value < 0x56bc75e2d63100000 
+----
+
+Transactions that emit a ValsEvent(...) event with an array with a first element equal to 5
+
+[source,jsx]
+----
+// Event Signature: ValsEvent(uint256[3] vals)
+vals[0] == 5
+----
+
+Transactions that invoke a greet(...) function with an unnamed string of "hello"
+
+[source,jsx]
+----
+// Function Signature: greet(address, string)
+$1 == "hello"
+----
+
+[[notifications]]
+== Notifications
+
+When triggered, a Sentinel can notify one or more slack webhooks, email lists, or execute an autotask.
+
+=== Slack Configuration
+
+* *Name* is the name for this slack configuration.  For instance, you might name it the name of the channel.
+
+* *Webhook URL* is the URL from your slack management console to use for notification. 
```

Comment:
Let's ellaborate here on how to set up a webhook in Slack

Reply:
~ellaborate~ elaborate